### ISSUE TITLE
About Normalization

### ISSUE THREAD

**Issue by ZhouQianang** [ali-vilab/UniAnimate-DiT#22]
Hi, I have a small question. In the training script (`train_unianimate_wan.py`), the input frame is normalized using `v2.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])`. However, in the test script, there is no identical process; instead, a simpler normalization is applied in `pipe.preprocess_image`:
```
image = torch.Tensor(np.array(image, dtype=np.float32) * (2 / 255) - 1).permute(2, 0, 1).unsqueeze(0)
```
I am wondering whether these two processes are equivalent and whether the difference matters.

Thank you!

**Comment by lifeiteng** [comment]
答案是：没问题，这两个处理是分别针对 video 和 image 的，处理对象不同，所以处理方式不需要保持一致。

理由：
`v2.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])` 是处理 video 的，理论上它需要跟 3D VAE 训练阶段处理video保持一致；
在生成阶段，是不知道的 video的，video 是生成的目标，所以不需要有什么 video 处理。

推理阶段的 `image = torch.Tensor(np.array(image, dtype=np.float32) * (2 / 255) - 1).permute(2, 0, 1).unsqueeze(0)` 需要跟 training_step 里的实现保持一致 `random_ref_dwpose_data = self.randomref_embedding_pose((batch["random_ref_dwpose_data"]/255.).to(torch.bfloat16).to(self.device).permute(0,3,1,2)).unsqueeze(2) # [1, 20, 104, 60]`。
